src/screens/ENSConfirmRegisterSheet.tsx

```diff
@@ -126,7 +126,10 @@ export default function ENSConfirmRegisterSheet() {
   const avatarImage =
     avatarMetadata?.path || initialAvatarUrl || params?.externalAvatarUrl || '';
   const { result: dominantColor } = usePersistentDominantColorFromImage(
-    avatarImage
+    avatarImage,
+    {
+      signUrl: true,
+    }
```

Comment:
Does every usage require signing? Can we just sign every time and remove this option?

Reply:
I changed it